### PR TITLE
[#26] Update current sponsors

### LINKED ISSUE #29: Update current sponsors
The only current sponsor is https://thinkst.com/  - $5000 per year. Many thanks

Variomedia was a sponsor in 2015

Azure was a sponsor for some time...  but they have just removed our Azure account without notice ...  we still somehow get help from MS via GitHub services

### PR DESCRIPTION
**Pull request by david-gi**
Fixes #29 

- Added premium tier banner
- Restyled current sponsor list with premium and standard tiers (identified by size and row position)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -581,7 +606,51 @@ <h4>Join the
...
+          <div class="col">
+            <p>
+              Participate in the
+              <a href="https://github.com/twisted/trac-wiki-archive/blob/trunk/TwistedSoftwareFoundation.mediawiki">
+                Twisted Project Sponsorship Program</a>!
+            </p>
+            <p>
+              For
+              <a href="https://github.com/twisted/trac-wiki-archive/blob/trunk/TwistedSoftwareFoundation.mediawiki#Sponsorship_Levels">
+                Silver Sidewinder and higher-level</a> sponsors, we will display your logo here on the front page for one year.
```
Comment (adiroiban) [inline index.html:646]:
```suggestion
              For donations greater than $400 per month, we will display your logo at the top of the page.
              For donations greater than $200 per month, we will display your logo on this page.
              Check <a href="https://github.com/sponsors/twisted/">
                GitHub Sponsors</a> for more information about sponsoring perks.
```

**Review by adiroiban — APPROVED** [review]
Changes looks good. Many thanks for the help here. Much appreciated.

I will commit a few changes

**Comment by adiroiban** [comment]
@glyph if you have time, please take a look at the updated website

We use GitHub pages for production.

For PR preview, I setup a netlify deployment 

You can check it here

https://deploy-preview-33--twisted-pr-preview.netlify.app/

**Comment by glyph** [comment]
Thanks for setting up the preview! Looks good.

**Review by glyph — APPROVED** [review]

**Pull request merged**